PowershellTools.Common/ServiceManagement/DebuggingContract/DebuggerStoppedEventArgs.cs

```diff
@@ -22,17 +22,21 @@ public class DebuggerStoppedEventArgs
         [DataMember]
         public int Column { get; set; }
 
+        [DataMember]
+        public bool OpenScript { get; set; }
+
         public DebuggerStoppedEventArgs()
         {
             this.BreakpointHit = false;
         }
 
-        public DebuggerStoppedEventArgs(string script, int line, int column)
+        public DebuggerStoppedEventArgs(bool breakpointHit, string script, int line, int column, bool openScript)

```

Comment:
Per convo re: ordering/default values, and Comments on what each item does, especailly when not self-evident.